.github/workflows/RELEASE_ISSUE.yml

```diff
@@ -19,6 +19,50 @@ jobs:
         template-path: 'docs/.project/RELEASE_TEMPLATE.md'
         package-path: 'app/package.json'
 
+  createMilestone:
+    needs: createReleaseIssue
+    if: github.event.issue.milestone
```

Comment:
Shouldn't this be enclosed in `${{ }}`?

Reply:
For `if` clauses, using the expression syntax is optional (cf. https://docs.github.com/en/actions/learn-github-actions/expressions).
We can use if for consistency. I think it looks cleaner without it, especially for multi-line conditions